### ISSUE TITLE
Remove `Communications Complete` date from Engagements

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-api-v3#1892]
The `Communications Complete` date is no longer needed on Engagements. 

We can remove this from the UI, API and DB. 
This data does not need to be retained.

Coordinate with Data Analytics (primarily Ken Jones) when removing this from the DB and DOMO API.

**Comment by blakekrammes** [comment]
UI: https://github.com/SeedCompany/cord-field/pull/869
API: https://github.com/SeedCompany/cord-api-v3/pull/1957
Procedures: https://github.com/SeedCompany/cord-procedures/pull/14

**Comment by kjones140** [comment]
@sethmcknight 

Here's a suggestion to  improve the process between DevOps and the DA team when fields are removed.
The DA team should schedule removal of the field from Domo first in their sprint, and then after that has been completed DevOps can plan in their next sprint to remove the field from the DB and the Domo API.

Planning it in this order will prevent any waiting between the teams.

For this issue, I recommend the following:  I will create the CommunicationsCompleteDate field on the Domo side and fill it with NULLs.  I will no longer pull that field from the LanguageEngagement or InternEngagment API.  This way, the DA team can plan in a future sprint to remove this field from the 9 cards and 40 datasets that it is currently a part of.

**Comment by blakekrammes** [comment]
Ken confirmed via slack that this can be merged when ready.

**Comment by sethmcknight** [comment]
Thanks for the suggestion @kjones140, I think that's a great idea and will work to coordinate and prioritize in that way moving forward.

> @sethmcknight
> 
> Here's a suggestion to improve the process between DevOps and the DA team when fields are removed.
> The DA team should schedule removal of the field from Domo first in their sprint, and then after that has been completed DevOps can plan in their next sprint to remove the field from the DB and the Domo API.
> 
> Planning it in this order will prevent any waiting between the teams.
> 
> For this issue, I recommend the following: I will create the CommunicationsCompleteDate field on the Domo side and fill it with NULLs. I will no longer pull that field from the LanguageEngagement or InternEngagment API. This way, the DA team can plan in a future sprint to remove this field from the 9 cards and 40 datasets that it is currently a part of.

**Comment by sethmcknight** [comment]
@blakekrammes we can merge to production whenever you are ready.

> Ken confirmed via slack that this can be merged when ready. "The new posts API and removal of communicationscompletedate s/b ok to push whenever."

**Comment by blakekrammes** [comment]
We're blocked by the seed-api deployment failure unfortunately.

**Comment by CarsonF** [comment]
This is live in cord field and seed api. Merged to develop on cord api, not yet live.

**Comment by blakekrammes** [comment]
Property has been deleted from the DB.
